openhtmltopdf-pdfbox/src/main/java/com/openhtmltopdf/pdfboxout/PdfBoxUserAgentFactory.java

```diff
@@ -0,0 +1,95 @@
+package com.openhtmltopdf.pdfboxout;
+
+/**
+ *  PdfBoxUserAgentFactory. Singleton.
+ */
```

Comment:
Oh no! Please no singletons....

Reply:
Singleton pattern appeared a long before Guice, Spring iOC, Weld, Dagger, etc. 
 "Singletons should only be @Singleton annotated and injected classes."
who has said such a stupid thing? 
Singletons should only be @Singleton annotated in CDI / DI Enviroments, where dependency injection container is used, without CDI container, you can still use and write Singletons and where are normal classes which have static fields-like behavior.

"does not fit the current design"
LOL. I do not see any design at all.